index.css

```diff
@@ -1,11 +1,338 @@
 body{
     margin: 0;
-    background-color: #BAC7A7;
+    font-family: Arial, Helvetica, sans-serif;
+    background-color: #1e1e1e;
+    color: #ffffff;
 }
 
 
 .navBar{
     display: flex;
-    flex-direction: row;
-    background-color: #698474;
-}
\ No newline at end of file
+    background-color: #151515;
+    justify-content: center;
+    padding: 1%;
+    
+}
+
+
+
+.child-navBar{
+    margin-left: 30%;
+}
+
+.img-logo{
+    color: #ffffff;
+    width: 25px;
+    height: 25px;
+}
+
+
+a{
+    text-decoration: none;
+}
+
+.home{
+    margin-right: 30px;
+    cursor: pointer;
+}
+
+.about{
+    cursor: pointer;
+}
+
+.contact{
+    margin-left: 30px;
+    margin-right: 30px;
+    cursor: pointer;
+}
+
+.registration{
+    cursor: pointer;
+}
+
+.login{
+    margin-left: 30px;
+    margin-right: 30px;
+    cursor: pointer;
+}
+
+
+.section1{
+    display: flex;
+    justify-content: center;
+    background-image: url(https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcTPs8YFUpAZT7dIXbQwGE243-DzvqhKoI251Q&usqp=CAU);
+    /* background-color: #111111; */
+
+}
+
+.img-pp{
+    color: #ffffff;
+    width: 370px;
+    height: 370px;
+    margin-right: 2%;
+    
+}
+
+
+.intro{
+    margin-right: 16%;
+    margin-top: 8%;
+    margin-left: 6%;
+    
+}
+
+
+.name{
+    font-size: xx-large;
+    margin-top: 1%;
+    margin-bottom: 1%;
+}
+
+
+h1{
+    color: #ffffff;
+    
+}
+
+h5{
+    color: #ffffff;
+    
+}
+
+h4{
+    color: #ffffff;
+    
+}
+
```

Comment:
h1, h5 and h4 have similar styles. A more concise way of styling them is:
h1, h5, h4 {
   color: #ffffff
}